### DISCUSSION TITLE
How can I chat with the entire Codebase?

### DISCUSSION THREAD

**Discussion by PierrunoYT** [community/community#75366] [Copilot Conversations]
### Select Topic Area

Question

### Body

I want to chat with the entire Codebase not only the file which is opened. I tried to use @workspace /fix but it only use the file which I have opened.

**Comment by Akash1134** [comment]
Hi @PierrunoYT ,

I have labelled this discussion as a product feedback so our engineers can capture it. To share feedback about GitHub Copilot Chat, you can use the share feedback button in the GitHub Copilot Chat window.

- Scroll to the top of the GitHub Copilot Chat window.
- Click the share feedback button.
- To open an issue to request a feature, next to Feature request for GitHub Copilot Chat, click Get started.

**Reply by PierrunoYT** [reply]
Thank you. Do I need to submit again?
